App/Containers/ThreadsList.js

```diff
@@ -17,6 +18,13 @@ import PreferencesActions from '../Redux/PreferencesRedux'
 import TextileNodeActions from '../Redux/TextileNodeRedux'
 import { getThreads } from '../Redux/PhotoViewingSelectors'
 
+function isEmpty (obj) {
```

Comment:
Don't think this is needed. See comment below.

Reply:
Yup you're right, removed.